From 10c0b6abc74642243c7bc290308e2c7a833b7fe6 Mon Sep 17 00:00:00 2001
From: Zhen Chen <chenzhen@rock-chips.com>
Date: Mon, 28 May 2018 17:30:15 +0800
Subject: [PATCH] arm: dts: rockchip: modify 'gpu_power_model' for Midgard DDK
 r18 on rk3288

The values of the coefficients are the ones in px30.dtsi,
according to Rocky Hao.

Change-Id: I2bcd19f4d9bbddcb02a6ecae743b212778c00178
Signed-off-by: Zhen Chen <chenzhen@rock-chips.com>
---
 arch/arm/boot/dts/rk3288.dtsi | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index d9f027b95ddf..e6ca69ba8ee1 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -1682,10 +1682,8 @@
 
 		gpu_power_model: power_model {
 			compatible = "arm,mali-simple-power-model";
-			voltage = <950>;
-			frequency = <500>;
-			static-power = <300>;
-			dynamic-power = <396>;
+			static-coefficient = <411000>;
+			dynamic-coefficient = <733>;
 			ts = <32000 4700 (-80) 2>;
 			thermal-zone = "gpu-thermal";
 		};
-- 
2.35.3

